spyder/plugins/ipythonconsole/tests/test_ipythonconsole.py

```diff
@@ -92,6 +92,13 @@ def __getattr__(self, attr):
     # Tests assume inline backend
     CONF.set('ipython_console', 'pylab/backend', 0)
 
+    # Start in a new working directory the console
+    new_wdir = 'new_workingdir/'
```

Comment:
Maybe it is better to define this as a CONSTANT outside so that it can be used later at the
```
CONSTANT = 'new_workingdir/'
...

assert act_wdir == CONSTANT
```

Reply:
This also needs to be handled with a pytest marker because it's modifying the fixture. Please see how we handle the `use_introspection` marker in the `main_window` fixture, present in `spyder/app/tests/test_mainwindow.py`.